packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_model.dart

```diff
@@ -144,6 +146,7 @@ class LoggingTableModel extends DisposableController
           ),
         ],
         ToggleFilter<LogDataV2>(
+          id: 'gc-logs',
```

Comment:
Why not have all const strings like this defined in one location?

Reply:
stored the ids as consts on their respective classes.